pkg/backends/newrelic/flush.go

```diff
@@ -23,7 +23,7 @@ type timeSeries struct {
 }
 
 // addMetric adds a metric to the series.
-func (f *flush) addMetric(n *Client, metricType string, value float64, persecond float64, hostname string, tags gostatsd.Tags, name string) {
+func (f *flush) addMetric(n *Client, metricType string, value float64, persecond float64, tags gostatsd.Tags, name string) {
```

Comment:
We don't need the hostname here? Is it just being pushed into the tags?

Reply:
It's never consumed at all, probably copied and pasted from the DD backend which does give it special usage (which is probably why the notion exists at all).

Reply:
Sounds good to me.